projects/XSEDE_ECSS.yaml

```
@@ -0,0 +1,12 @@
+Name: XSEDE_ECSS 
+Description: Support for XSEDE users through ECSS
+Department: San Diego Supercomputing Center
+FieldOfScience: Computer Science
+Organization: University of California, San Diego
+PIName: Robert Sinkovits
+
+ID: '714'
```

```suggestion
ID: '715'
```

There was another project PR sitting in our queue so we've got to update the ID. @matyasselmeci and I have been talking about what it'd take to remove this field so hopefully we can avoid this in the future!